### PR TITLE
out_http: Add descriptions about AWS Signature Version 4 support

### PR DESCRIPTION
**Pull request by daipom**
https://docs.fluentd.org/output/http

New feature of Fluentd v1.17.0.
From https://github.com/fluent/fluentd/pull/4459

### CODE REVIEW

File: output/http.md [hunk 1]
```diff
@@ -275,30 +288,65 @@ Specifies HTTP authentication:
...
 The password for basic authentication.
 
+#### Parameters for `method aws_sigv4`
+
+Parameters for [AWS Signature Version 4](https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html).
+
+```text
+<auth>
+  method aws_sigv4
+  aws_service osis
```
Comment (kenhys) [inline output/http.md:314]: Does this example `osis` is appropriate?  I'm not sure `osis` is famous enough.

Comment (daipom) [reply output/http.md:314]: Thanks for your review!

I brought this example from https://github.com/fluent/fluentd/issues/4444.
I can't judge whether this is appropriate.
If you have a better example, please let me know.

Comment (kenhys) [reply output/http.md:314]: Thanks.
In AWS context, it may be obvious.
okay, so keep it as is.

**Review by kenhys — APPROVED** [review]
LGTM.

File: output/http.md (hunk 1 above)
Comment (daipom) [reply output/http.md:314]: OK! Thanks!

**Pull request merged**